### DISCUSSION TITLE
Package via Git URL

### DISCUSSION THREAD

**Discussion by kenorb** [FirstGearGames/FishNet#253] [Feature Request]
Is it possible to add this package via Git URL, instead of Unity Store?
Normally you've to add `package.json` file into this repo, so it can be recognized as package.

**Comment by kenorb** [comment]
Ok, my understanding is that isn't a package as such, just an asset library. So I'm going to clone it as submodule into the existing project, that would work.

**Comment by FirstGearGames** [comment]
I may be wrong but it's my understanding I cannot implement this without changing the entire structure of the project.
